plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/my-subscriptions.tsx

```diff
@@ -38,13 +39,24 @@ export default function MySubscriptions(): JSX.Element {
 		},
 		''
 	);
-	const updateConnectionHTML = sprintf(
-		// translators: %s is a link to the update connection page.
+
+	const installedTableDescription = createInterpolateElement(
```

Comment:
Nice, didn't know you can do that.

Reply:
It's pretty powerful. @andfinally showed me this.